src/commands/ask.ts

```diff
@@ -1,17 +1,30 @@
-import { ChatInputCommandInteraction, SlashCommandBuilder } from 'discord.js';
+import { SlashCommandBuilder } from '@discordjs/builders';
+import { APIChatInputApplicationCommandInteraction, InteractionResponseType, Routes } from 'discord-api-types/v10';
 import { getDefaultEmbed } from '../utils/embeds.js';
+import { Env, JsonResponse } from '../index.js';
+import { REST } from '@discordjs/rest';
+import { Command } from '../types';
```

Comment:
nit: Could we order imports with deps first, then local files? Does apply to all files.

Reply:
the format step should take care of this once we merge to main